apis/r/R/utils-tiledb.R

```diff
@@ -7,7 +7,7 @@ tiledb_zstd_filter <- function(level = 3L) {
 }
 
 match_query_layout <- function(layout) {
-  layouts <- c("ROW_MAJOR", "COL_MAJOR", "GLOBAL_ORDER", "UNORDERED")
+  layouts <- c("ROW_MAJOR", "COL_MAJOR", "GLOBAL_ORDER", "UNORDERED", "auto") # "auto" is used by libtiledbsoma
```

Comment:
Good catch! Can you also remove GLOBAL_ORDER and UNORDERED, since those aren't valid values for soma_array_reader?